virtual_workstation_setup.sh

```diff
@@ -12,6 +12,27 @@ silent_grep() {
     command grep -q  > /dev/null 2>&1 "$@"
 }
 
+set +e
+for filename in ~/.ssh/id_*; do
+    [[ "$filename" == *".pub" ]] && continue
+    # check if SSH key has passphrase
+    ssh-keygen -y -P "" -f "$filename" > /dev/null 2>&1
```

Comment:
This is to support a new `openssh` format of the keys: https://security.stackexchange.com/a/129729

Reply:
Nice. Also, nice use of `ssh-keygen`